bobtemplates/odoo/security/security/acl_+security.model.name_underscored+.xml.bob

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright {{{ copyright.year }}} {{{ copyright.name }}}
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+{{% if odoo.version < 9 %}}
+<openerp>
+<data>
+{{% else %}}
+<odoo>
+{{% endif %}}
+
+    <record model="ir.model.access" id="{{{ security.model.name_underscored }}}_access_name"> <!-- TODO acl id -->
+        <field name="name">{{{ security.model.name_dotted }}} access name</field> <!-- TODO acl name -->
+        <field name="model_id" ref="model_{{{ security.model.name_underscored }}}"/>
```

Comment:
We should review this value because we are not sure the model_id is in this module. I would suggest to add a TODO here